backends/cuda-shared/ceed-cuda-shared.c

```diff
@@ -40,8 +45,13 @@ static int CeedInit_Cuda_shared(const char *resource, Ceed ceed) {
 
   Ceed_Cuda_shared *data;
   ierr = CeedCalloc(1,&data); CeedChk(ierr);
+  data->Q1d  = -1;
```

Comment:
These are at the `CEED` level?

Reply:
I think this is where they make the most sense, since this tells which basis is currently loaded in the constant memory, however it's buggy and doesn't bring any performance gain, I'll remove it.